src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -554,8 +566,13 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     }
     else
     {
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+        interval_min = BLEMgrImpl().Internal_Slow_Advertising_MIN;
+        interval_max = BLEMgrImpl().Internal_Slow_Advertising_MAX;
```

Comment:
You can use the actual constants here. There is no need to access members that are only relevant to the implementation and come from a global config header.